alertmanager/multitenant.go

```diff
@@ -41,11 +85,70 @@ var (
 		Help:      "Time spent requesting configs.",
 		Buckets:   prometheus.DefBuckets,
 	}, []string{"operation", "status_code"})
+	statusTemplate      *template.Template
+	allConnectionStates = []string{"established", "pending", "retrying", "failed", "connecting"}
 )
 
 func init() {
 	prometheus.MustRegister(configsRequestDuration)
 	prometheus.MustRegister(totalConfigs)
+	statusTemplate = template.Must(template.New("statusPage").Funcs(map[string]interface{}{
+		"printState": func(enabled bool) string {
+			if enabled {
+				return "enabled"
+			}
+			return "disabled"
+		},
+		"printConnectionCounts": func(conns []mesh.LocalConnectionStatus) string {
+			counts := make(map[string]int)
```

Comment:
style nit: using `make` only when it's really needed (for chans or slices) seems to have become a more common style in Go projects. So rather `counts := map[string]int{}`.